src/tools/rbd_mirror/Replayer.cc

```diff
@@ -801,5 +696,65 @@ void Replayer::handle_pre_release_leader(Context *on_finish) {
   on_finish->complete(0);
 }
 
+void Replayer::handle_image_acquire(const std::string &global_image_id,
+                                    const instance_watcher::ImagePeers& peers) {
+  dout(20) << "global_image_id=" << global_image_id << dendl;
+
+  Mutex::Locker locker(m_lock);
+
+  assert(peers.size() == 1U);
```

Comment:
Nit: I assume this is temporary -- why not directly pass the full collection through?